src/ScrollView/scrollView.scss

```diff
@@ -0,0 +1,113 @@
+@import '~gemini-scrollbar/gemini-scrollbar.css';
+@import '~sass-flex-mixins/src/sass-flex-mixins.scss';
+
+$scrollbarSize: 7px !default;
+$scrollbarSizeBig: 9px !default;
```

Comment:
where is this variable used? the name is not very clear... (I thought we had two available themes: standard/big)